.github/workflows/CI.yml

```diff
@@ -15,6 +19,15 @@ jobs:
       - name: Setup
         run: |
           make fetch-thirdparty
+      - name: Install solc
+        run: |
+          sudo add-apt-repository ppa:ethereum/ethereum
+          sudo add-apt-repository ppa:ethereum/ethereum-dev
+          sudo apt-get update
+          sudo apt-get install solc
+      - name: Build
+        run: |
+          cargo build
```

Comment:
Let's move this `make build` ... new entry to Makefile, so we have all commands in single place